backend/core/settings/prod.py

```
@@ -0,0 +1,64 @@
+    # Fallback to a random key so that ``check --deploy`` and other
+    # management commands executed in CI still have a secure value available.
+    # Real deployments should still provide the key through an environment
+    # variable to maintain session persistence.
+    return secrets.token_urlsafe(64)
+
+
+SECRET_KEY = _resolve_secret_key()
+
+
+DEFAULT_ALLOWED_HOSTS = ["apatie.example"]
+ALLOWED_HOSTS = ENV.list("ALLOWED_HOSTS", default=DEFAULT_ALLOWED_HOSTS)
```

Ruff (0.13.3)

`ENV` may be undefined, or defined from star imports

(F405)